Source/src/components/ComponentCamera.cpp

```diff
@@ -103,7 +104,10 @@ void Hachiko::ComponentCamera::OnTransformUpdated()
 {
     ComponentTransform* transform = game_object->GetTransform();
 
-    frustum.SetFrame(transform->GetGlobalPosition(), transform->GetFront(), transform->GetUp());
+    frustum.SetPos(transform->GetGlobalPosition());
```

Comment:
Does separating this in different calls bring any benefit?

Reply:
It seemingly led to less random exceptions from MathGeoLib, but can be coincidence. I saw it on Strawhat Engine and tried in Hachiko to see if it makes any difference.

Reply:
No big difference probably so its ok to leave it like this. Its weird that it affects consistency tho